lib/generateMarkdown.js

```diff
@@ -72,8 +75,9 @@ function generateMarkdown(options) {
             // TODO: display their schema
         }
 
-        // Render section for each property
+        // Render section for each property & definition
         md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings);
```

Comment:
I think you'd want to remove this one since you're calling it below with the extra parameter?